.github/workflows/test-release.yaml

```
@@ -0,0 +1,45 @@
+name: Create test release
+on:
+  workflow_dispatch:
+    inputs:
+       version:
```

I think this is over-indented.
```suggestion
      version:
```

---

Fixed